README.md

```diff
@@ -106,14 +106,14 @@ build.
    # Install build tools needed for Rust. If you're building a 32-bit compiler,
    # then replace "x86_64" below with "i686". If you've already got git, python,
    # or CMake installed and in PATH you can remove them from this list. Note
-   # that it is important that you do **not** use the 'python2' and 'cmake'
-   # packages from the 'msys2' subsystem. The build has historically been known
-   # to fail with these packages.
+   # that it is important that you do **not** use the 'cmake' packages from
+   # the 'msys2' subsystem. The build has historically been known to fail with
```

Comment:
Does this not hold for the python3 package from the msys2 subsystem? Maybe leave it in for now, since presumably it would?

Reply:
I'm pretty sure it still holds.
Packages from msys2 subsystem are build for unix so `python` will tell it's running on unix based OS.